CHANGES.md

```diff
@@ -11,6 +11,9 @@
 
 ### Fixed
 
+- Fixed parsing the changelog if it is written in the style of
+  keepachangelog.com(https://keepachangelog.com/) (#421, @ifazk)
```

Comment:
Could you move this to the "added" section? I don't think it is a fix, because the fact that it didn't use some specific format wasn't "broken", it was just unsupported. Now you added support :-)

The markdown syntax for the link is wrong, it needs `[keepachangelog.com](https://keepachangelog.com/)`